ldaptor/ldiftree.py

```diff
@@ -282,30 +282,30 @@ def deleteChild(self, rdn):
         return defer.maybeDeferred(self._deleteChild, rdn)
 
     def __repr__(self):
-        return '%s(%r, %r)' % (self.__class__.__name__,
+        return '{}({!r}, {!r})'.format(self.__class__.__name__,
                                self.path,
                                self.dn.getText())
 
     def __lt__(self, other):
         if not isinstance(other, LDIFTreeEntry):
             # We don't return NotImplemented so that we get the same
             # result in Python2 and Python3.
-            raise(TypeError, 'unorderable types: %r > %r' % (self, other))
+            raise(TypeError, 'unorderable types: {!r} > {!r}'.format(self, other))
         return self.dn < other.dn
 
     def __gt__(self, other):
         if not isinstance(other, LDIFTreeEntry):
-            raise(TypeError, 'unorderable types: %r < %r' % (self, other))
+            raise(TypeError, 'unorderable types: {!r} < {!r}'.format(self, other))
         return self.dn > other.dn
 
     def commit(self):
-        assert self.path.endswith(u'.dir')
-        entryPath = self.path[:-len(u'.dir')]
+        assert self.path.endswith('.dir')
+        entryPath = self.path[:-len('.dir')]
         d = defer.maybeDeferred(_putEntry, entryPath, self)
 
         def eb_(err):
             from twisted.python import log
-            log.msg("[ERROR] Could not commit entry: {0}.".format(self.dn))
+            log.msg("[ERROR] Could not commit entry: {}.".format(self.dn))
```

Comment:
Do we really need this change? 

```
$ python3
Python 3.8.2 (default, Jul 16 2020, 14:00:26) 
>>> '{} {}'.format('bla', 'na')
'bla na'
```

This can be reverted and the coverage test will not complain.

The best fix for this is to write a new test which checks this code.

Reply:
test added

Reply:
test added